clang/test/Preprocessor/riscv-target-features-sifive.c

```diff
@@ -0,0 +1,95 @@
+// RUN: %clang --target=riscv32 \
+// RUN: -march=rv32i_zve32x_xsfmm128t -x c -E -dM %s \
```

Comment:
indent continuation lines by 2 spaces.

Remove unneeded `-x c`

Reply:
This pattern exists in many places in clang/test/Preprocessor/riscv-target-features.c. Should we fix those?

Reply:
I guess many of them have been fixed already, but there are still a few left.